sentry.server.config.js

```diff
@@ -0,0 +1,19 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN
+const NODE_ENV = process.env.NODE_ENV
+
+Sentry.init({
+  dsn: SENTRY_DSN || 'https://67fc84f1cafe443c8bb9d73eb172712a@o225462.ingest.sentry.io/4504001045790720',
```

Comment:
Question: Should we use a different DSN for Next.js client and Next.js server?

Reply:
Discussed with Roland. We decided to keep it simple for now and use a single DSN. Then we can improve afterwards.